obspy/core/inventory/channel.py

```diff
@@ -333,6 +333,8 @@ def plot(self, min_freq, output="VEL", start_stage=None, end_stage=None,
             amplitude/phase spectrum. If not specified, a new figure is opened.
         :type unwrap_phase: bool
         :param unwrap_phase: Set optional phase unwrapping using NumPy.
+        :type deg: bool
+        :param deg: if True plot bode in degrees
```

Comment:
two back ticks for formatting:
```
``True``
```
in docstring will be displayed by sphinx as monotyped:

``True``